src/Component.hpp

```diff
@@ -634,6 +634,11 @@ class Node : public Component {
     ~Node() override = default;
 #ifdef PROC_CPUINFO
 public:
+    /**
+     * @private
```

Comment:
does this @private mean it is not included in the API documentation? if yes, this call should be public

Reply:
Yes, fixed!